### ISSUE TITLE
1.81.0 Broke indentation styles

### ISSUE THREAD

**Issue by MariuzM** [microsoft/vscode#189783]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.81.0
- OS Version: Mac

Steps to Reproduce:

Before this update in settings i had 
```
"editorIndentGuide.activeBackground": "#798092",
"editorIndentGuide.background": "#00000000",
```

Now with this new update i cant use `"editorIndentGuide.background": "#00000000",` with value 00000000 because it does not show the `798092`

The only way i can see that line now is if i enable other lines but i dont want that, i want like it was before so i dont see red lines and only see green line 

<img width="1019" alt="image" src="https://github.com/microsoft/vscode/assets/36937039/13512ee9-dad8-41e2-9f88-ddf7e2302432">

Here i enable my style that hides the line because of last 00 alpha values but it hides everything now...
<img width="966" alt="image" src="https://github.com/microsoft/vscode/assets/36937039/3a719f8e-e443-4262-93a0-18a1d967e238">

**Comment by hediet** [comment]
Can you use some other color with 0 alpha?

**Comment by MariuzM** [comment]
without alpha i can match background color but then problem is when selecting it looks ugly, it shows now the dark colors when before it was clien because of alpha channel 

<img width="93" alt="image" src="https://github.com/microsoft/vscode/assets/36937039/8e546f44-0760-4aed-8178-0d629e54cf19">

**Comment by hediet** [comment]
> without alpha

I mean set the alpha channel to 0 so that it is fully transparent, but use a different color than black. E.g. `#ffffff00`

**Comment by MariuzM** [comment]
Not working, hm, recap i want like vscode has done before i was using 
```
    "editorIndentGuide.activeBackground": "#798092",
    "editorIndentGuide.background": "#00000000",
```


I have tried, but this wont make any difference alpha channel now for `editorIndentGuide.background1` disabled color for `editorIndentGuide.activeBackground1` 
```
    "editorIndentGuide.activeBackground1": "#798092",
    "editorIndentGuide.background1": "#ffffff00",
```

The closest is to set the to match editor background color 
<img width="1061" alt="image" src="https://github.com/microsoft/vscode/assets/36937039/943dbb38-7957-4376-a880-48e14e9f0c1e">


But that then gives issue when selecting 
<img width="445" alt="image" src="https://github.com/microsoft/vscode/assets/36937039/3a3edc61-fa36-4f79-8210-c24a1a2a023b">

**Comment by MariuzM** [comment]
Just tried setting alpha like so     "editorIndentGuide.background1": "#22262e01",
and this worked basically 01, but when you set 00 then it kills the `activeBackground1`

**Comment by zpincus** [comment]
This is correct: `editorIndentGuide.background1` with any color and alpha of `00` still disables `activeBackground1` (as of 1.84.2); however setting this to any other alpha, even just `01`, restores the previous behavior (which was pretty nice insofar as allowing one to see the active indents but not having all the visual clutter elsewhere).

**Comment by olejorgenb** [comment]
I think this issue is obsolete? I can't find the `editorIndentGuide.background` option anymore.

https://github.com/microsoft/vscode/issues/197066 requests customization of the indent guide style.
